ci/post/main.py

```diff
@@ -145,12 +155,29 @@ def main():
 		print("ERROR: Invalid release mode %s passed. Expected release or nightly!" % sys.argv[1])
 		sys.exit(1)
 
+	# bail if any build subflows failed
+	success = ((os.environ["LINUX_RESULT"] == "success") and (os.environ["WINDOWS_RESULT"] == "success"))	##!< true if both linux and windows builds successful
+	if not success:
+		print("ERROR: One or more builds failed to publish!")
+		sys.exit(1)
```

Comment:
We used to post nightlies on Nebula and the forum even if one of the builds failed. Is this change intentional?

Reply:
Right now this script only works on the Releases, the Nightlies are still working in their own script on the nightlybuild repo.  I'd have to confer with ya'll on Discord to see what the build failure policy is.